powertools-sqs/src/main/java/software/amazon/lambda/powertools/sqs/SqsBatch.java

```diff
@@ -32,6 +32,19 @@
  * {@link SqsBatch#suppressException()} to true. By default its value is false
  * </p>
  *
+ * <p>
+ * If you want certain exceptions to be treated as permanent failures, i.e. exceptions which are not worth retrying and
+ * want such message should be moved to configured dead letter queue of the source SQS queue, you can use
```

Comment:
"these can be immediately moved to the dead letter queue associated to the source SQS queue" 

Link to DLQ dev guide - https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-dead-letter-queues.html